CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 project(MeasurementKit)
-cmake_minimum_required(VERSION 3.0.0)
+cmake_minimum_required(VERSION 2.8.0)
```

Comment:
I lowered the CMake minimum version because apparently this yields more useful warnings.